_kbarticles/2021-05-20-connecting-to-brokered-service-instances.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: "Connecting to brokered Elasticsearch service instances"
+date: May 20, 2021
+excerpt: Some suggestions for connecting to brokered Elasticsearch instances
```

Comment:
@ChrisMcGowan @bengerman13 Would love your eyes on this article to ensure I didn't miss anything.